ivy/functional/frontends/paddle/tensor/tensor.py

```diff
@@ -167,6 +167,13 @@ def erf(self, name=None):
     def subtract(self, y, name=None):
         return paddle_frontend.Tensor(ivy.subtract(self._ivy_array, _to_ivy_array(y)))
 
+    @with_unsupported_dtypes(
+        {"2.5.1 and below": ("float16", "uint8", "int8", "bool")}, "paddle"

```

Comment:
could you please just leave it as "float32", "float64" for now, thanks

Reply:
I have tested on my local machine by importing paddle and noticed that PyTorch's `subtract` (also `subtract_`) method supports every other data type including `["int16", "int32", "int64", "bfloat16", "float32", "float64", "complex64", "complex128"]` and does not support `["float16", "uint8", "int8", "bool"]` so that's why I have included them in `@with_unsupported_dtypes` decorator.
 
So I suggest to make the above method exactly similar to the PyTorch's `subtract_` method.
Thank you very much! 😊